lib/src/judo_floating_card.dart

```diff
@@ -0,0 +1,210 @@
+part of judo.components;
+
+enum MessageLevel { error, warning, info, success }
+
+abstract class FloatingMessage {
+  String get actionCallName;
+  String get message;
+  MessageLevel get messageLevel;
+}
+
+class JudoFloatingCard extends StatelessWidget {
+  const JudoFloatingCard({
+    Key key,
+    this.messageStack = const [],
+    @required this.closeAction,
+  }) : super(key: key);
+
+  final List<FloatingMessage> messageStack;
+  final Function closeAction;
+  final double height = 144;
```

Comment:
Add these 2 as optional constructor params with the current values as default please, so that in the future if we'd need to adjust the dimensions we won't need to create another release of this package!